pkg/apk/apkindex.go

```diff
@@ -127,9 +127,13 @@ func ParsePackageIndex(apkIndexUnpacked io.Reader) ([]*Package, error) {
 		case "U":
 			pkg.URL = val
 		case "D":
-			pkg.Dependencies = strings.Split(val, " ")
+			if val != "" {
```

Comment:
Please add a comment that such split on empty string would return `[""]` that is not what we want `[]`

Reply:
Done.